themes/src/main/resources/theme/keycloak/common/resources/lib/patternfly4/tile.css

```diff
@@ -0,0 +1,202 @@
+/*Internet Explorer 11 compatibility workaround - IE does not support CSS variables */
```

Comment:
If I understand it correctly, this directory is a 3rd party library. I'm not sure we should be adding custom files in there. It might even get deleted on some dependencies update. Or am I missing something? @ssilvert

Reply:
@vmuzikar it isn't from a 3rd party library, it's actually a part of the most recent releases of PF4 [1][2] and it's just copied from there - purely just for the sake of the transition phase. 

This file can be deleted as soon as `@patternfly` module will be updated in the project. 

[1] https://www.patternfly.org/v4/components/tile
[2] https://github.com/patternfly/patternfly/blob/84ae888cf912693b209ac0cf8f63b3ab3ddf59c2/src/patternfly/components/Tile/tile.scss

Reply:
edit: sry @vmuzikar, I misunderstood your input at first, to clarify that - it's just partially custom, the vast majority of the code comes from PF4 [1] and the IE optimization part is actually the only custom piece of code. In other words, we could eventually move just the IE workaround part from there (e.g. to login.css). But just a suggestion ;).

[1] https://github.com/patternfly/patternfly/blob/84ae888cf912693b209ac0cf8f63b3ab3ddf59c2/src/patternfly/components/Tile/tile.scss

[]

Reply:
Thanks for the clarification. I still see it as a custom CSS file (manually added, edited, not managed by NPM, etc.) which as such shouldn't be placed among other (managed) dependencies. But let's wait for @ssilvert, maybe I'm wrong.

Reply:
Btw, why not to put these changes into `login.css` alongside other IE hacks?